datarobot_batch_scoring/detect.py

```diff
@@ -0,0 +1,91 @@
+import math
+import re
+
+
+class Detector(object):
```

Comment:
it is obvious, cause there is no calls to `chardet` and it used at the point, when we already know data encoding (:. 
Just delimiters. Also, it's main method return value called `delimiter`.